data/gegno/gegno intro missions.txt

```diff
@@ -439,6 +442,311 @@ mission "Giaru Gegno: Revisit"
+				flee
+
+
+			label "approach"
+			action
+				set "scin: entered epidote"
+			`	Walking towards the Scin, you notice the freigher is indeed large - maybe four times the size of a Hauler, or almost two times the size of a Behemoth. As you approach it, an elevator lowers from underneath, and a group of Scin await your arrival. Their attire is much different than that of the Vi, most of them wearing what looks like segmented armor with many layers of plates, giving off the appearance of scales. One of them, however, has a much lighter armor set focused mainly around their shoulders, with orange robe-like garments underneath, and are slightly thinner than the other Scin. Although none of them are nowhere near as bulky as the Vi, they all still appear slightly taller than a human. More importantly, they seem far more eager to interact with you, seeing that they are comfortable with you being near their ship.`
+			`	Once you're at the elevator, the Scin in heavier armor escort you onboard, and elevator lifts back into the ship, leading into some sort of spacious internal bay. The Scin in the robe walks off nearby and grabs some sort of device, moving it over to where you stand, and gestures you to join them.`
+			choice
+				`	(Join them.)`
+				`	(Reach for your pistol.)`
+					goto "pistol"
+			`	You walk over to the device, getting a good look at the robed Scin. Their skin appears almost bone-like with very faint patterns of light green. Unlike the Vi, they have three small horns and barely any hair at all. They seem just as interested in your appearance, looking you up and down. After messing with the device a bit more, the Scin reveals some sort of tablet or drawing device.`
+			`	The Scin then reaches their palm out and makes a mark on one half of the display, then turns it to you.`
+			choice
+				`	(Place your hand on the device.)`
+				`	(Do nothing.)`
+					goto "images"
+			`	Placing your own palm on the device, you feel it vibrate a little. A detailed wireframe of your hand appears on the other half of the device. The Scin moves the device aside, then extends one arm out showing the top of their hand.`
+			choice
+				`	(Extend your arm as well.)`
+				`	(Do nothing.)`
+					goto "images"
+			action
+				set "scin: gestures"
+			`	Reaching with your right hand, the Scin leans forward and carefully examines it, then makes another note on their device. After writing down a few more things, they turn to you gesture strangely, crossing their arms together in a plus-shaped fashion towards their left. The other heavier-armored Scin move closer, escort you back to the elevator and back to your ship, but along the way are a few Scin you haven't seen yet. Their armor is much different than that of the others, characterized by darker shades, sharper edges, and several gadgets, but what is most notable is their possession of sickle-esque blade weapons.`
```

Comment:
I'm not entirely sure if this is what you're going for, but the first clause should clarify who is doing the reaching.
```suggestion
			`	As you reach out with your right hand, the Scin leans forward and carefully examines it, then makes another note on their device. After writing down a few more things, they turn to you and gesture strangely, crossing their arms together in a plus-shaped fashion towards their left. The other heavier-armored Scin move closer and escort you back to the elevator and to your ship, but along the way, you notice a few Scin you haven't previously seen. Their armor is much different than that of the others, characterized by darker shades, sharper edges, and several gadgets, but what is most notable is their possession of sickle-like blade weapons.`
```